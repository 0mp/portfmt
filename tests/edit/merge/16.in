LDFLAGS+=	-L${FOO}
DOS2UNIX_FILES=	foo
