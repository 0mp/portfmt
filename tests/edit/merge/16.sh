${PORTEDIT} merge -e 'LDFLAGS_i386=-Wl,-znotext' 16.in | diff -L 16.expected -L 16.actual -u 16.expected -
