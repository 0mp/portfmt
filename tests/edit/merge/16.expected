LDFLAGS+=	-L${FOO}
LDFLAGS_i386=	-Wl,-znotext
DOS2UNIX_FILES=	foo
